hs_core/templates/503.html

```diff
@@ -94,9 +94,16 @@
                         <a href="#" class="dropdown-toggle" data-toggle="dropdown">{{ request.user }} <b class="caret"></b></a>
                         <ul class="dropdown-menu">
                             <li class="account">
-                                <p><b>{{ request.user }}</b>{% if request.user.email %}<br />{{ request.user.email }}{% endif %}</p>
-                                <p><a href="{% url 'profile_update' %}">Profile</a> | <a href="{% url 'logout' %}" id="signout-menu">Sign out</a>
-                                <div class="clearfix"></div>
+                                {% if request.user.userprofile.picture and request.user.userprofile.picture.url%}
+                                <div style="background-image: url('{{ request.user.userprofile.picture.url }}'); float:left;"
+                                     class="round-image profile-pic-thumbnail">
+                                </div>
+                                {% endif %}
+                                <span>
+                                    <strong>{{ request.user }}</strong>
+                                    {% if request.user.email %}<br/>{{ request.user.email }}{% endif %}
+                                </span>
+                                <span><a href="/user/{{ user.id }}/">Profile</a> | <a href="{% url 'logout' %}" id="signout-menu">Sign out</a></span>
```

Comment:
You mean issue https://github.com/hydroshare/hydroshare/issues/724 - refactor genericresource.html, right? If so, It'd be good to add a note to that issue for bookkeeping purpose so it will not be forgotten.